src/modules/Contracts.ts

```diff
@@ -168,17 +169,12 @@ export class Contracts {
   ): Promise<any> {
     const {
       blockNumber,
-      confirmations,
-      confirmationType,
-      gasMultiplier,
-      gas, // don't send gas
-      gasPrice, // don't send gasPrice
       ...otherOptions
-    } = {
+    } = this.toCallOptions({
       ...this.defaultOptions,
       ...specificOptions,
-    };
-    return (method as any).call(otherOptions);
+    });
+    return (method as any).call(otherOptions, blockNumber);
```

Comment:
What's this do, passing in blockNumber?

Reply:
It allows you to make calls at a certain block number instead of at the latest block. Helpful for two use-cases:
1. getting older data from the chain, say you want it from the past
2. ensuring that when you do several node calls in parallel, they all return data from the same block instead of defaulting to 'latest' where you may get data from different blocks due to race condition